app/models/form.rb

```diff
@@ -29,4 +31,8 @@ def draft_belonging_to(user)
   def draft_belonging_to?(user)
     draft_belonging_to(user).present?
   end
+
+  def form_name
+    [name.parameterize].join('-')
```

Comment:
parameterize already returns what we need. The joins is unnecessary here. I'll remove it.